source/backend/api/Areas/Projects/Models/ProjectSearchModel.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+
+namespace Pims.Api.Areas.Projects.Models
+{
+    public class ProjectSearchModel
```

Comment:
This model looks pretty similar to the existing ProjectModel. It would be better if we have only one given the data is pretty much the same

Reply:
Deleted model